modules/publicize/ui.php

```diff
@@ -281,7 +296,23 @@ function admin_page() {
 							e.preventDefault();
 							return false;
 						}
-					})
+					});
+
+					// deprecation tooltip
+					var setup = function() {
+						$('#jetpack-gplus-deprecated-notice').first().pointer(
+							{
+								content: "<h3>Google Plus support is being removed<\/h3><p>Google recently <a href=\"https://www.blog.google/technology/safety-security/expediting-changes-google-plus/\">announced</a> that Google Plus is shutting down in April 2019, and access via third-party tools like Jetpack will cease in March 2019.<\/p><p>For now, you can still post to Google Plus using existing connections, but you cannot add new connections. The ability to post will be removed in early 2019.<\/p>","position":{"edge":"right","align":"bottom"},
```

Comment:
This would ideally be translateable. We may also want to use "Google+" instead of "Google Plus"

Reply:
1e59e0158c96edca81f95fe5a4a2f7eee8002491 should fix this.